logi/logi.go

```diff
@@ -12,35 +12,35 @@ type WLogger interface {
 
 // Logger represents common interface for logging functionality
 type Logger interface {
-	// WithContext will return logger with filled basic fields (email, company, corelationID)
-	WithContext(ctx context.Context) Logger
+	// With returns logger with basic fields based on context and your custom fields
+	With(ctx context.Context, keysAndValues ...any) Logger
 
-	// Errorf logs a templated message with the provided error
-	Errorf(format string, err interface{}, args ...interface{})
+	// Err is a shorter version of With(ctx, "error", err)
```

Comment:
Not sure I understand this comment, Err is **not** same as With because it doesn't take ctx, right? If I want to print ctx AND error, I understand I should call both With() and Err().
Maybe something like this is more clear?
```suggestion
	// Err returns logger with the provided error
```